tests/performance/reindexing/reindexing_and_feeding.rb

```diff
@@ -193,21 +155,75 @@ def get_reindexing_status
     return status['status'].first
   end
 
+  def graph_config
+    {
+      'reindex' => {
+	'reindexing.throughput'   => { :y_min =>  10000, :y_max =>  50000 },
+	'reindexing.time.seconds' => { :y_min =>     30, :y_max =>    120 }
+      },
+      'feed' => {
+	'qps' =>                     { :y_min =>  10000, :y_max =>  50000 },
```

Comment:
Is it feed throughput you want here? If so, _feeder.throughput_ is the key used when generating the report.

Reply:
Thanks, yes. `qps` is used by fbench :)